tools/compare/compare.py

```diff
@@ -8,16 +8,55 @@
 
 
 def main() -> None:
-    new_path = Path("Rb_v1.1")
-    old_path = Path("Rb_v1.0")
+    new_path = Path("misc_v1.1")
```

Comment:
In principle, I am fine with hardcoding the versions in this helper script. However, I would add a well-visible comment that one has to change this code if one likes to compare databases for other versions.